doc/source/conf.py

```diff
@@ -44,11 +45,35 @@
     'sphinx_gallery.gen_gallery',
 ]
 
+# -- sphinx_gallery configuration
+from sphinx_gallery.gen_gallery import ExplicitOrder
+
+sg_gallery_dir = 'auto_examples'
+sg_examples_dir = '../../examples'
+
+def subsection_title_order(src_dir: str):
+    """
+    src_dir : str
+        string name of source path
+    """
+
+    sub_folders = [p for p in Path(src_dir).iterdir() if p.is_dir()]
+    final_order = []
+    for sub_folder in sub_folders:
+        if sub_folder.name == "release_highlights":
+            final_order.insert(0, sub_folder)
+        else:
+            final_order.append(sub_folder)
+    return ExplicitOrder(final_order)
+    
 sphinx_gallery_conf = {
-     'examples_dirs': '../../examples',   # path to your example scripts
-     'gallery_dirs': 'auto_examples',  # path to where to save gallery generated output
+     'examples_dirs': [sg_examples_dir],   # path to your example scripts
+     'gallery_dirs': [sg_gallery_dir],  # path to where to save gallery generated output
+      #'subsection_order' : subsection_title_order(sg_examples_dir), # TODO@DAVID: turns out unneeded
```

Comment:
We could have a part in the user guide on the doc and internals, to document these perhaps? e.g. a developer guide.

Reply:
That sounds like a great idea